pkg/client/vault/client.go

```diff
@@ -102,23 +133,66 @@ func (c *Client) CreateVault() (*CreatedVault, error) {
 	}
 
 	return &CreatedVault{
-		ID:  didKey,
-		KMS: kmsLoc,
-		EDV: edvLoc,
+		ID: didKey,
+		Authorization: Authorization{
+			KMS: &Location{
+				URI:  kmsURI,
+				ZCAP: kmsZCAP,
+			},
+			EDV: edvLoc,
+		},
 	}, nil
 }
 
-// CreatedVault represents success response of CreateVault function.
-type CreatedVault struct {
-	ID  string    `json:"id"`
-	EDV *Location `json:"edv"`
-	KMS *Location `json:"kms"`
+// SaveDoc saves a document by encrypting it and storing it in the vault.
+func (c *Client) SaveDoc(vaultID, id string, content interface{}, auth *Authorization) (*SavedDoc, error) {
+	const encryptionKeySizeBytes = 32
+
+	encContent, err := encryptContent(c.webKMS(vaultID, auth.KMS), c.webCrypto(vaultID, auth.KMS),
+		random.GetRandomBytes(encryptionKeySizeBytes))
```

Comment:
replace `random.GetRandomBytes(encryptionKeySizeBytes)` with `content` and remove line 149 above (`const encryptionKeySizeBytes = 32`)